sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/models/Aggregation.java

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.developer.loadtesting.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+/**
+ * Aggregation type.
+ */
+public final class Aggregation extends ExpandableStringEnum<Aggregation> {
```

Comment:
With the introduction of models, all the README samples should be updated to use the new APIs that use models. These are much simpler to use than BinaryData based APIs.

Reply:
Updated the README and the samples